paymentsheet/src/main/java/com/stripe/android/link/ui/wallet/WalletViewModel.kt

```diff
@@ -132,9 +133,15 @@ internal class WalletViewModel @Inject constructor(
                 if (paymentDetailsState.paymentDetails.isEmpty()) {
                     navigateAndClearStack(LinkScreen.PaymentMethod)
                 } else {
+                    val currentState = _uiState.value
                     _uiState.update {
                         it.updateWithResponse(paymentDetailsState.paymentDetails)
                     }
```

Comment:
Nit: You can use `updateAndGet` here.

Reply:
https://github.com/stripe/stripe-android/pull/11267/commits/693fd0df006835cd3e8126e8891f5c4526b67b0f